### DISCUSSION TITLE
What should I credit if I use code from here

### DISCUSSION THREAD

**Discussion by Shluffy** [theforumhelpers/theforumhelpers.github.io#277] [Q&A]
@scratch-bookmarklets github site repo credits the org. how should i credit you? i use some quickreply css

**Comment by leahcimto** [comment]
If I read the license file correctly, you don't need to give credit. (I'm not sure on that though, so pinging @Accio1)If you were to give credit you would probably give it to the org though.

**Comment by Accio1** [comment]
> @Scratch-Bookmarklets github site repo credits the org. how should i credit you? i use some quickreply css

Technically you don't have to give credit since it is an open source site. However, credit is always good and crediting the organization is a good way to do it.
